### PR TITLE
ci: Finch Windows MSI tool

### PR DESCRIPTION
**Pull request by KevinLiAWS**
Issue #, if available:

*Description of changes:*
MSI tool to generate Windows installer
*Testing done:*
Tested with github action and locally

- [X] I've reviewed the guidance in CONTRIBUTING.md

#### License Acceptance

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: msi-builder/postinstall.bat [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@echo off
+SET InstallDir=%~1
+SET FilePath=%InstallDir%\os\finch.yaml
+
+if exist "%FilePath%" (
+    powershell -Command "$installPath = '%InstallDir%'.Replace('\', '/'); (Get-Content '%FilePath%') -replace '__INSTALLFOLDER__', $installPath | Set-Content '%FilePath%'"
```
Comment (pendo324) [inline msi-builder/postinstall.bat:6]: I don't see where `__INSTALLFOLDER__` is set. Should there be a manual step to add it to `os\finch.yaml` before starting to build the installer?

Comment (pendo324) [reply msi-builder/postinstall.bat:6]: Thanks. We should probably still add that to the README just in case we ever need to do a manual build

**Review by pendo324 — APPROVED** [review]
LGTM pending one addition to the readme

File: msi-builder/postinstall.bat (hunk 1 above)
Comment (KevinLiAWS) [reply msi-builder/postinstall.bat:6]: Thanks. I will update the ReadMe with the auto build PR.

**Pull request merged**